src/components/ValidationMessage/ValidationMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import CorrectValidation from './CorrectValidation'
```

Comment:
I believe this might be over-engineering. This approach's not wrong, componentization has been done nicely, but I wonder if it's necessary. 

Could you have used the `message-background` div on both the `CorrectValidation` and `ErrorValidation` components instead? Even though on paper looks good, having an entry point for two different components might add complexity to the application.

I would recommend directly use the `CorrectValidation` and `ErrorValidation` separately